test/UnitTests/MSTest.CoreAdapter.Unit.Tests/Helpers/DictionaryHelperTests.cs

```diff
@@ -37,42 +29,45 @@ public void ConcatenatingDictionariesReturnsEmptyDictionaryWhenBothSidesAreNullO
         var actual = source.ConcatWithOverwrites(overwrite, nameof(source), nameof(overwrite));
         var expected = new Dictionary<string, string>();
 
-        actual.Should().BeEquivalentTo(expected);
+        actual.ToList().Sort();
+        expected.ToList().Sort();
+        Verify(actual.SequenceEqual(expected));
     }
 
-    [TestMethod]
     public void ConcatenatingDictionariesReturnsSourceSideWhenOverwriteIsNullOrEmpty()
     {
         var source = new Dictionary<string, string>
         {
-            ["aaa"] = "source",
             ["bbb"] = "source",
+            ["aaa"] = "source",
```

Comment:
Any reason to change the order here?

Reply:
This applies to other tests below as well.

Reply:
yes, because I use the sort function just to be sure It works correct